packages/core/src/Stores/client-store.js

```diff
@@ -60,6 +60,7 @@ export default class ClientStore extends BaseStore {
         payment_agent_withdraw: '',
     };
     @observable account_limits = {};
+    @observable login_history = {};
```

Comment:
Could this state be kept "locally" in the `<Loginhistory />` component instead of globally in the client store? Or is there a specific reason as to why we need this in the `client-store`?

Reply:
I don't see any reason. I think same applies to `account limits` too.